pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1309,13 +1309,13 @@ type QueueConfig struct {
 	// MaxSamplesPerSend is the maximum number of samples per send.
 	MaxSamplesPerSend int `json:"maxSamplesPerSend,omitempty"`
 	// BatchSendDeadline is the maximum time a sample will wait in buffer.
-	BatchSendDeadline string `json:"batchSendDeadline,omitempty"`
+	BatchSendDeadline *Duration `json:"batchSendDeadline,omitempty"`
```

Comment:
should be marked as optional
```suggestion
	// +optional
	BatchSendDeadline *Duration `json:"batchSendDeadline,omitempty"`
```